src/components/conjunction/ConjunctionObjectData.tsx

```diff
@@ -15,8 +14,8 @@ type ConjunctionObjectDataProps = {
   secondaryObject: TypeSatelliteOut | null;
 };
 
-const ConjunctionObjectData = ({ primaryObject, secondaryObject }: ConjunctionObjectDataProps) => {
-  const t = useTranslations('Accordions.Conjunction.object_data');
+const ConjunctionObjectData = async ({ primaryObject, secondaryObject }: ConjunctionObjectDataProps) => {
+  const t = await getTranslations('Accordions.Conjunction.object_data');
```

Comment:
Same, this should not be in Accordion translations

Reply:
Fixed